armi/tests/tutorials/c5g7-blueprints.yaml

```diff
@@ -70,33 +181,51 @@ blocks:
             isotopics: mox low
             Tinput: 25.0
             Thot: 25.0
-            od:  1.08
+            od: .8190
             latticeIDs: [ML]
         mox medium fuel:
             shape: Circle
             material: UO2
             isotopics: mox medium
             Tinput: 25.0
             Thot: 25.0
-            od:  1.08
+            od: .8190
             latticeIDs: [MM]
         mox high fuel:
             shape: Circle
             material: UO2
             isotopics: mox high
             Tinput: 25.0
             Thot: 25.0
-            od:  1.08
+            od: .8190
             latticeIDs: [MH]
+        void 1:
```

Comment:
I actually wasn't sure if i should define multiple void 1 components for each type of mox fuel to link to. Thoughts?

Reply:
I see the authors you're using did use the term void but I think it will be more clear to most readers if you actually call this a "gap". 

Mostly I think this has become just non-trivial enough that the non-obvious components would be better off with 1-liner comments saying things like `# gap between fuel and clad for the MOX pins` No strong opinion on 1 or 2 gaps.

Reply:
gap is fine, i just thought it would be better to use the terminology used in the documentation.

Can you go into a little more detail of the pros and cons of having 1 gap for each type of mox fuel, vs 1 for all mox fuel. For example, if there is only 1 for all the mox fuel, it seems like the gaps wouldn't be able to change dimensions independently for different fuel types. Is that right??